### DISCUSSION TITLE
Hydra release cycle

### DISCUSSION THREAD

**Discussion by igorrohal** [ory/hydra#2508] [Q&A] [answered]
Hello Hydra Community! 👋 

May I ask you what the release cycle of Hydra versions is? According to the release history, there isn't any regular schedule (e.g. 1 per month, 1 per quarter, etc.).

I appreciate all the info!
Igor

**Comment by vinckr** [comment, marked as answer]
You are correct, there is no regular schedule. 
Have a look at the upcoming milestones:
https://github.com/ory/hydra/milestones

When all issues are resolved / PR's merged for a milestone, that version will be released.

**Reply by igorrohal** [reply]
Thanks very much for the info!
